packages/ndla-ui/src/Embed/conceptComponents.tsx

```diff
@@ -182,20 +184,21 @@ export const ConceptNotionV2 = forwardRef<HTMLDivElement, ConceptNotionProps>(
     ref,
   ) => {
     const { t } = useTranslation();
-
+    const isGloss = conceptType === 'gloss';
```

Comment:
Denne brukes kun med ! foran seg så kunne forsåvidt vore invertert til `notGloss = conceptType !== 'gloss` for å gjøre det litt enklere å lese, men egentlig berre pirk.

Reply:
Eventuelt `isConcept`?

Reply:
Det er også en mulighet :)